infra/prod/generate.yaml

```diff
@@ -21,6 +21,8 @@ steps:
     args:
       - 'clone'
       - '--depth=1'
+      - '--single-branch'
```

Comment:
I don't think this is needed here, as we're specifying `depth`. That said, it does no harm.